README.md

```diff
@@ -101,23 +104,25 @@ Chewy is compatible with MRI 2.5-3.0¹.
 
 See [Migration guide](migration_guide.md).
 
-## Usage
+### Compatibility with Active Record
 
-### Client settings
+`5.2, 6.0, 6.1` Active Records versions are supported by all Chewy versions.
 
-There are two ways to configure the Chewy client:
+## Usage
```

Comment:
It might be a good idea to add some free form introduction text here, that outlines how Chewy is used and what steps the users will have to take to set it up. I find it a bit weird to that we immediately jump to the client settings.